apps/omg_eth/lib/omg_eth/root_chain/abi_function_selector.ex

```diff
@@ -82,4 +82,103 @@ defmodule OMG.Eth.RootChain.AbiFunctionSelector do
       ]
     }
   end
+
+  # min_exit_period/0, get_version/0, exit_games/0, vaults/0 are
+  # victims of unfortinate bug: https://github.com/poanetwork/ex_abi/issues/25
+  # All these selectors were intially pulled in with
+  # `ABI.parse_specification(contract_abi_json_decoded,include_events?: true)`
+  # and later modified so that `types` hold what `returns` should have because of
+  # issue 25.
+  # the commented properties of the struct is what it was generated,
+  # the new types were added to mitigate the bug.
+  def min_exit_period() do
```

Comment:
these selectors are our API to contracts

Reply:
I think the long term plan is to do the following:
`ex_plasma - is a thin protocol specification for transaction construction and validation.`

`ex_plasma-contracts - is a thin API for plasma-contract exposed functions and events.` 
But to achieve that, we need to work on ExABI a little bit, hopefully that clears in the following months. 
At the moment, the amount of bugs ExABI has, it would be of little value to put it in a separate library. 
Also, ex_plasma_contracts is made up name. Not sure yet how we should call it.